core/bundles/link/lib/pool/form/students.ex

```diff
@@ -1,17 +1,95 @@
 defmodule Link.Pool.Form.Students do
   use CoreWeb.UI.LiveComponent
 
+  alias Core.Accounts
+  alias Core.Enums.StudyProgramCodes
+
+  alias EyraUI.Text.Title2
+  alias CoreWeb.UI.ContentListItem
+
   prop(user, :any, required: true)
+  data(students, :map)
+
+  def update(_params, socket) do
+    students =
+      Accounts.list_students([:profile, :features])
+      |> Enum.map(&to_view_model(&1, socket))
+
+    {
+      :ok,
+      socket
+      |> assign(students: students)
+    }
+  end
+
+  defp to_view_model(%{
+    email: email,
+    inserted_at: inserted_at,
+    profile: %{
+      fullname: fullname,
+      photo_url: photo_url
+    },
+    features: features
+  }, socket) do
+
+    subtitle =
+      [email] ++ get_study_programs(features)
```

Comment:
`[email|get_study_programs(features)]`  would be slightly nicer (saves allocating a list).

Reply:
Aaah ja die vergeet ik steeds! 👍